### PR TITLE
fix: disable cache for v1 mode in sample test

### LINKED ISSUE #6640: [bug] sample v2 test enables cache for all pipelines with v1 mode
### What happened:
sample v2 test enables cache for all pipelines with v1 mode
### What did you expect to happen:
sample v2 test should disable cache for all pipelines with v1 mode
### Environment:

*  How do you deploy Kubeflow Pipelines (KFP)? 

*  KFP version: 

*  KFP SDK version: 

### Labels

---

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

### PR DESCRIPTION
**Pull request by capri-xiyue**
**Description of your changes:**
Fixed #6640 
disable cache for v1 mode in sample test
**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: samples/test/util.py [hunk 1]
```diff
@@ -535,3 +539,14 @@ def _parse_parameters(execution: metadata_store_pb2.Execution) -> dict:
         if name.startswith('output:'):
             parameters['outputs'][name[len('output:'):]] = raw_value
     return parameters
+
+
+def disable_cache():
+    def _disable_cache(task):
```
Comment (chensun) [inline samples/test/util.py:545]: If you don't need to parameterize anything, you don't need the outer function. 
See this for example: https://github.com/kubeflow/pipelines/blob/master/samples/core/pipeline_transformers/pipeline_transformers.py

Comment (capri-xiyue) [reply samples/test/util.py]: done

**Comment by capri-xiyue** [comment]
Screen Shot 2021-09-28 at 9 48 28 PM
v1 pipeline does not use cache

**Pull request merged**